cpp/tests/binaryop/binop-compiled-test.cpp

```diff
@@ -820,4 +822,24 @@ TEST_F(BinaryOperationCompiledTest_NullOpsString, NullMin_Vector_Vector)
   CUDF_TEST_EXPECT_COLUMNS_EQUIVALENT(expected, result->view());
 }
 
+TEST(BinaryOperationCompiledTest, LargeColumnNoOverflow)
+{
+  cudf::size_type num_rows{1799989091};
```

Comment:
Recommend separating the digits for better readability.
```suggestion
  cudf::size_type num_rows{1'799'989'091};
```

Reply:
Thanks, good idea